repo/repo.go

```diff
@@ -721,14 +690,7 @@ func (h *Handler) deleteBlob(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if err := os.Remove(path); err != nil {
-		if h.opt.Debug {
-			log.Print(err)
-		}
-		if os.IsNotExist(err) {
-			httpDefaultError(w, http.StatusNotFound)
```

Comment:
It looks like this code was incorrect all the time. A `DELETE` should be idempotent, which means that it can be reissued and get the same response. In this case `ErrNotExist` is fine and should be ignored, returning a 200.

Reply:
I've added a commit to fix this behavior. But, I'm wondering whether it would be a good idea to split that into a separate PR.